ProcessMaker/Http/Controllers/TaskController.php

```diff
@@ -11,12 +11,16 @@ public function index()
   {
       return view('tasks.index');
   }
-//     public function edit(Tasks $tasks) /*<-----*/ // 
-//   {
-//     return view('tasks.edit',compact($tasks));  //is the edit page added to routes?
-//   }
-//   public function create()
-//   {
-//     return view('tasks.create');
-//   }
+    public function edit(Tasks $tasks) /*<-----*/ // 
+  {
+    return view('tasks.edit', ["task"=>$task]);  //is the edit page added to routes?
```

Comment:
This doesn't work. The variables have different names. Also, use the compact method instead of any array.

https://www.easylaravelbook.com/blog/passing-multiple-variables-into-a-laravel-5-view/

Reply:
Where is the test?